### ISSUE TITLE
Navigation menu block: can not move keyboard focus to the 'Add Navigation Link' inserter button.

### ISSUE THREAD

**Issue by enriquesanchez** [WordPress/gutenberg#20741]
As part of the accessibility test performed by the Accessibility team (#20369) we found that keyboard-only navigation in the navigation menu block was lacking.

One of the most critical issues was the inability to move keyboard focus to the 'Add Navigation Link' inserter button:

![2020-03-09 14-16-04 2020-03-09 14_19_22](https://user-images.githubusercontent.com/3276087/76253844-066d1680-6211-11ea-9cb8-9f96c7b0eeb1.gif)

In the example above, after focusing on the 'Sample page" item, focus skips the inserter button and instead jumps to the sidebar. Similarly, shift-tab returns focus from the sidebar to the 'Sample page' item, skipping the inserter button.

**Comment by enriquesanchez** [comment]
Perhaps a quick solution for this would be to have the 'Add link' label also show up even if you only have one menu item:

<img width="556" alt="Screen Shot 2020-03-09 at 14 42 25" src="https://user-images.githubusercontent.com/3276087/76255710-44b80500-6214-11ea-8df9-7b2479a712a6.png">

**Comment by mtias** [comment]
Would this be the same as https://github.com/WordPress/gutenberg/issues/20728 ?

**Comment by enriquesanchez** [comment]
It's very similar, yes. 

In this particular case, moving keyboard focus to the inserter button was not possible. In addition, only after adding more than one menu item, the 'Add link...' label appears.

Once this label is present then keyboard focus works as expected.

**Comment by tellthemachines** [comment]
Closing this as it has been fixed by #37637. 

Note: if "contain text cursor inside block" is enabled, it still won't work, but that's tracked separately as part of #36710, as it requires finding an alternative to Arrow key navigation.
